utils/event-schemaorg.ts

```diff
@@ -0,0 +1,52 @@
+export interface EventSchemaOrg {
```

Comment:
I believe that we're not importing this interface anywhere else, so we can remove the export:

```suggestion
interface EventSchemaOrg {
```